src/components/UploaderForm.tsx

```diff
@@ -0,0 +1,98 @@
+import { FunctionComponent, useState, useMemo } from 'react';
+import { Button, ErrorMessage } from 'lbh-frontend-react';
+import { Formik, Form } from 'formik';
+import * as Yup from 'yup';
+import UploaderPanel from './UploaderPanel';
+import { useRouter } from 'next/router';
+import { DocumentType } from '../domain/document-type';
+import { EvidenceRequest } from '../domain/evidence-request';
+
+const UploaderForm: FunctionComponent<Props> = (props) => {
+  const router = useRouter();
```

Comment:
Should this component have a dependency on router? It should probably receive the request ID as a prop

Reply:
it started up in the page - i moved it down because the form probably needs to do some navigation on success?

Reply:
It should just call a success callback—the component that uses this form will handle what happens upon success 👍